Content/default/SAFE.App.sln

```diff
@@ -31,6 +31,8 @@ Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Server.Tests", "tests\Serve
 EndProject
 Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Client.Tests", "tests\Client\Client.Tests.fsproj", "{CF8C20AC-1FC4-4114-ACB8-6CE6E51F34F9}"
 EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Build", "Build.fsproj", "{C0244434-EC34-4D74-93A9-0E00DEC0C4D1}"
```

Comment:
Line 8, `build.fsx` doesn't exist anymore

Reply:
That's `Build.fsproj` not `build.fsx`

Reply:
I was referring to the line 8 of this file that I can't comment on: https://github.com/SAFE-Stack/SAFE-template/pull/415/files/0251d8cdf2b26618d86b66320283c87564e2f7ba#diff-f564918bd20243543596c1f4ec459f93c958aa325432596895e431f9a15f126aL8

As this was done: https://github.com/SAFE-Stack/SAFE-template/pull/415/files/0251d8cdf2b26618d86b66320283c87564e2f7ba?file-filters%5B%5D=.fsx#diff-a26cc857ceabaa57f1ac087600547fd4cfd4db7c28c641d264378d2d5de4f294L1

Or maybe I understood something wrong

Reply:
Yes you're right. Can't comment on L8 due to https://github.com/isaacs/github/issues/1655

Reply:
I've addressed this here: https://github.com/SAFE-Stack/SAFE-template/pull/443

Reply:
This has been fixed